src/mapml.js

```diff
@@ -2505,72 +2829,75 @@ M.TemplatedTileLayer = L.TileLayer.extend({
       // at this time.  In the case of multiple <coordinates> per geometry, we
       // will look for class attribute on the first <coordinates> element.
       // var cl; // classList -> DOMTokenList https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList
-      switch (geometry.firstElementChild.tagName.toUpperCase()) {
-        case 'POINT':
-          coordinates = [];
-          geometry.getElementsByTagName('coordinates')[0].textContent.split(/\s+/gim).forEach(parseNumber,coordinates);
-          pt = this.coordsToPoint(coordinates, tileCoords);
-          renderPoint(pt, geometry);
-          break;
-        case 'MULTIPOINT':
-          coordinates = [];
-          // TODO the definition of multipoint geometry was modified in testbed 15
-          // to align with geojson a bit better.  
-          // this modification requires one <coordinates> element with 1 or more
-          // text string coordinate pairs, per the model for <coordinates> in a
-          // linestring (but with different semantics). As such, in order to separately
-          // select and style a coordinate, the user has to wrap it in a <span class="...>
-          // The code below does not  support that yet.  the renderPoint code
-          // will have to use treewalker (as the polygon code does), and copy the
-          // classes from the geometry element to each child svg element
-          // created i.e. onto the circle or path(s) that are created.
-          geometry.getElementsByTagName('coordinates')[0].textContent.match(/(\S+ \S+)/gim).forEach(splitCoordinate, coordinates);
-          members = this.coordsToPoints(coordinates, 0, tileCoords);
-          for(member=0;member<members.length;member++) {
-            // propagate the classes from the feature to each geometry
-            const g = geometry;
-            feature.classList.forEach(val => g.getElementsByTagName('coordinates')[0].classList.add(val));
-            renderPoint(members[member], g.getElementsByTagName('coordinates')[0]);
-          }
-          break;
-        case 'LINESTRING':
-          coordinates = [];
-          geometry.getElementsByTagName('coordinates')[0].textContent.match(/(\S+ \S+)/gim).forEach(splitCoordinate, coordinates);
-          renderLinestring(this.coordsToPoints(coordinates, 0, tileCoords), geometry);
-          break;
-        case 'MULTILINESTRING':
-          members = geometry.getElementsByTagName('coordinates');
-          for (member=0;member<members.length;member++) {
+      //if(this.isVisible){
```

Comment:
odd that this whole block is showing an update but there doesn't appear to be any changes, at least by visual inspection.

I was thinking about rendering and maybe a solution to the rendering issue and this essentially duplicate code will be to subclass L.SVG (if possible) and set our renderer as an option for classes that need to render vectors.  I will look into this a bit, would appreciate your insights.

Reply:
It was indented because I wanted to prevent rendering if the layer's isVisible property was false, I felt that this was the wrong approach as this method should not even been invoked if the layer's isVisible property was false. The indentation will be undone